[2025-05-31T13:44:35.866+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-31T13:44:35.881+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: PlanningEngine.inferInitialSchemas manual__2025-05-31T13:42:52.456293+00:00 [queued]>
[2025-05-31T13:44:35.889+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: PlanningEngine.inferInitialSchemas manual__2025-05-31T13:42:52.456293+00:00 [queued]>
[2025-05-31T13:44:35.890+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-05-31T13:44:35.904+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): inferInitialSchemas> on 2025-05-31 13:42:52.456293+00:00
[2025-05-31T13:44:35.910+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=104) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-31T13:44:35.911+0000] {standard_task_runner.py:72} INFO - Started process 105 to run task
[2025-05-31T13:44:35.912+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'PlanningEngine', 'inferInitialSchemas', 'manual__2025-05-31T13:42:52.456293+00:00', '--job-id', '3421', '--raw', '--subdir', 'DAGS_FOLDER/***DAG.py', '--cfg-path', '/tmp/tmpgt7v6l0i']
[2025-05-31T13:44:35.913+0000] {standard_task_runner.py:105} INFO - Job 3421: Subtask inferInitialSchemas
[2025-05-31T13:44:35.957+0000] {task_command.py:467} INFO - Running <TaskInstance: PlanningEngine.inferInitialSchemas manual__2025-05-31T13:42:52.456293+00:00 [running]> on host 87ce0d954e61
[2025-05-31T13:44:36.036+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='PlanningEngine' AIRFLOW_CTX_TASK_ID='inferInitialSchemas' AIRFLOW_CTX_EXECUTION_DATE='2025-05-31T13:42:52.456293+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-31T13:42:52.456293+00:00'
[2025-05-31T13:44:36.037+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-05-31T13:44:36.089+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-05-31T13:44:36.097+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-05-31T13:44:36.098+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=PlanningEngine, task_id=inferInitialSchemas, run_id=manual__2025-05-31T13:42:52.456293+00:00, execution_date=20250531T134252, start_date=20250531T134435, end_date=20250531T134436
[2025-05-31T13:44:36.127+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-31T13:44:36.147+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-31T13:44:36.150+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
